kokoro/config/build/presubmit/plucky_aarch64.gcl

```diff
@@ -0,0 +1,10 @@
+import '../common.gcl' as common
```

Comment:
I think the third party app integration tests and presubmits are missing in this PR. Is this correct ? I'm comparing with the previous addition of an Ubuntu distribution https://github.com/GoogleCloudPlatform/ops-agent/pull/1826.

Reply:
Yes, it's missing on purpose as it hasn't been fixed by ObserveIQ yet. Once ObserveIQ fixes the 3p tests, I will add the kokoro configs in a separate PR.

This PR only enables ops agent tests running on the new Distro.